fixit/rules/rewrite_to_literal.py

```diff
@@ -64,14 +64,30 @@ class RewriteToLiteralRule(CstLintRule):
         Invalid("tuple()", expected_replacement="()"),
         Invalid("list()", expected_replacement="[]"),
         Invalid("dict()", expected_replacement="{}"),
+        Invalid("dict(test=[])", expected_replacement='{"test": []}'),
+        Invalid(
+            "dict(test=[], other=())", expected_replacement='{"test": [], "other": ()}'
+        ),
+        Invalid(
+            "dict(test=[], other=1)", expected_replacement='{"test": [], "other": 1}'
+        ),
+        Invalid(
+            "dict(other=1, test=[1, 2, 3])",
+            expected_replacement='{"other": 1, "test": [1, 2, 3]}',
+        ),
+        Invalid("dict([], other=1)", expected_replacement='{"other": 1}'),
```

Comment:
Let's make this test case more useful via combining keys from the list and kwargs.
```suggestion
        Invalid("dict([('a', 1), ('b', 2)], other=1)", expected_replacement='{"a": 1, "b": 2, "other": 1}'),
```